### ISSUE TITLE
Torrent Tracker Not Working

### ISSUE THREAD

**Issue by Natrinicle** [kiwix/container-images#240]
Connecting to the https torrent tracker directly with curl yields the following

```shell
$ curl -vvv https://tracker.openzim.org:6969/announce
*   Trying 51.159.30.163:6969...
* Connected to tracker.openzim.org (51.159.30.163) port 6969 (#0)
* ALPN: offers h2
* ALPN: offers http/1.1
*  CAfile: /etc/ssl/certs/ca-certificates.crt
*  CApath: /etc/ssl/certs
* [CONN-0-0][CF-SSL] TLSv1.0 (OUT), TLS header, Certificate Status (22):
* [CONN-0-0][CF-SSL] TLSv1.3 (OUT), TLS handshake, Client hello (1):
* OpenSSL SSL_connect: SSL_ERROR_SYSCALL in connection to tracker.openzim.org:6969 
* Closing connection 0
curl: (35) OpenSSL SSL_connect: SSL_ERROR_SYSCALL in connection to tracker.openzim.org:6969 
```

Checking both trackers with https://newtrackon.com/ per https://github.com/kiwix/container-images/issues/79 yields the following

```
https://tracker.openzim.org:6969/announce | 51.159.30.163 | Rejected | HTTP timeout
udp://tracker.openzim.org:6969/announce | 51.159.30.163 | Rejected | Wrong response length while announcing: 8
```

Trying to add torrents from https://library.kiwix.org to Deluge shows `Error: timed out` as well as `Error: skipping tracker announce (unreachable)` but doesn't specify which is from udp and which is from https as the application only shows the FQDN and the status and since both hostnames are identical the error messages flip and flop back and forth. No peers are ever received and the torrent progress is stuck at 0%.

I'm currently using Debian Testing with Deluge 2.0.3-3.3 and OpenSSL 3.0.8-1.

I was recommended to submit an issue here after contacting contact@kiwix.org but this repo appears to be for container-images, please let me know if I should refile this issue in another repo or location.

**Comment by kelson42** [comment]
@Natrinicle I think this problem has been fixed yesterday. We use now https in place of https. Let me know otherwise.

**Comment by rgaudin** [comment]
Torrents were incorrectly listing the tracker as HTTPS while it doesn't support it.

Torrents now properly list HTTP.

Can you download the torrent again and retry ?

**Comment by kelson42** [comment]
It looks like a whitelisting problem....

**Comment by rgaudin** [comment]
Indeed the whitelist appears to not be refreshed. Investigating

**Comment by rgaudin** [comment]
Fixed in 796397b. I confirm this entry is now in the whitelist
